### PR TITLE
added some useful commands

### PR DESCRIPTION
**Pull request by yash3001**
Addressing #10  

Added some useful commands as of now. Will continue to work on this

### CODE REVIEW

**Comment by ghost** [comment]
Pls resolve the conflict

**Comment by yash3001** [comment]
Done

File: Linux.md [hunk 1]
```diff
@@ -1,26 +1,114 @@
...
+`du -sh <file/folder_name>`
+
+### <a name="killall"></a> Kill all the instances of a particular program
+`killall <program_name>`
+
+### <a name="listContentsOfAFile"></a> List the conents of a file in a numbered fashion
+`nl <file_name>`
+
+### <a name="displayWordCount"></a> Display the word count in a text file
+`wc -l <file_name>`
```
Comment (ghost) [inline Linux.md]: pls remove this command

File: Linux.md [hunk 2]
```diff
@@ -1,26 +1,114 @@
...
+
+### <a name="listRunningProcesses"></a> List running processes in a tree like format
+`pstree`
+
+### <a name="displayTimeAndDate"></a> Display the time and date on the top right corner of the terminal
```
Comment (ghost) [inline Linux.md]: pls remove this command

File: Linux.md [hunk 3]
```diff
@@ -1,26 +1,114 @@
...
-### Check IP to whitelist
+### <a name="checkIpToWhitelist"></a> Check IP to whitelist
 `curl ifconfig.me ; echo`
 
-### Search Bash history
+### <a name="outputGeographicalInfo"></a> Outputs Geographical Information, regarding an ip_address
+`curl ipinfo.io ; echo`
+
+### <a name="compareTwoTextFiles"></a> Compare two text files and show the differences
+`diff -y -w 50 <file1> <file2> --suppress-common-lines`
```
Comment (ghost) [inline Linux.md]: pls remove this command

**Review by ghost — CHANGES_REQUESTED** [review]

File: Linux.md (hunk 1 above)
Comment (yash3001) [reply Linux.md]: done

File: Linux.md (hunk 2 above)
Comment (yash3001) [reply Linux.md]: done

File: Linux.md (hunk 3 above)
Comment (yash3001) [reply Linux.md]: done

**Review by ghost — APPROVED** [review]

**Pull request merged**